src/js/components/Select/SelectContainer.js

```diff
@@ -39,9 +39,27 @@ const SelectOption = styled(Button)`
   width: 100%;
 `;
 
+const ClearButton = ({ clear, onClear, theme }) => {
+  const { label, position, render } = clear;
+  const align = position !== 'bottom' ? 'start' : 'center';
+  if (render) {
+    return render({ onClear });
+  }
+  return (
+    <Box flex={false}>
```

Comment:
Do we need this Box? I'm just comparing this to how SelectOption and OptionBox work for options.

Reply:
removed the box